java/ql/src/utils/model-generator/ModelGeneratorUtilsSpecific.qll

```diff
@@ -58,38 +58,38 @@ class TargetApi extends Callable {
     ) and
     isRelevantForModels(this)
   }
-}
 
-private string isExtensible(RefType ref) {
-  if ref.isFinal() then result = "false" else result = "true"
-}
+  private string isExtensible(RefType ref) {
```

Comment:
These predicates should not be members of `TargetApi`, as that means there is a Cartesian product with the implicit `this` parameter.